packages/cardhost/app/routes/cards.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class CardsRoute extends Route {
+  @service cssModeToggle
+  queryParams = { editingCss: {} }
+
+  model({ editingCss }) {
+    this.cssModeToggle.setEditingCss(editingCss);
```

Comment:
This feels like something that will bite us, but good for now until we figure out the real flow we want this to have